@kiva/kv-components/vue/KvCheckbox.vue

```diff
@@ -0,0 +1,116 @@
+<template>
+	<div>
```

Comment:
Since this div has only one child element, you could drop this div and use the label as the root element for this component

Reply:
I'd like to keep this wrapping div to keep things predicitable, since the label has a display class, `tw-inline-flex` on it. This could cause issues if you were trying to show/hide things responsively like
`<kv-checkbox class="sm:tw-hidden lg:tw-block" />`